examples/example_plugin/example_plugin/urls.py

```diff
@@ -45,4 +47,9 @@
         name="examplemodel_changelog",
         kwargs={"model": ExampleModel},
     ),
+    path(
+        "docs/",
+        RedirectView.as_view(url=f"{settings.STATIC_URL}example_plugin/docs/index.html"),
```

Comment:
My limited understanding is that simply prepending `settings.STATIC_URL` may not do what we want in all cases (such as with remote static-file storage) and we should instead be using something like `from django.templatetags.static import static; static("example_plugin/docs/index.html")` throughout instead.

Reply:
Okay, I can try that. I think I saw this used somewhere else. Might need to create a bug for that when I find it.